File: DAW1/Programació/programació_java/098_ActObjectes01/src/Cotxe.java

public class Cotxe {
	
	//Constructor
	//Serveix per inicialitzar l'objecte, y es el primer que fará, per exemple per cridar base de dades etc..
	public Cotxe() {
		System.out.println("....creant l'objecte....");
	}
	
	//Definició atributs
		private String sMarca, sModel, sColor, sMatricula;
		
		//SETTERS Info que envíes
		public void setMarca(String marca) {
			sMarca = marca;
		}
		public void setModel(String model) {
			sModel = model;
		}
		public void setColor(String color) {
			sColor = color;
		}
		
		//----
		
		public void matricular() {
			int num = (int) ((Math.random() * 10000));
			sMatricula = String.valueOf(num);
			int[] lletres = new int [3];
			for (int i = 0; i < lletres.length; i++) {
				lletres[i] = (int) (Math.random()*(90 - 65 + 1) + 65);
			}
			
			for (int i = 0; i < lletres.length; i++) {
				sMatricula = sMatricula + String.valueOf(Character.toChars(lletres[i]));
			}
			
			
		}
		
		//GETTERS Info que rebs
		public String getMarca() {
			return sMarca;
		}
		
		public String getModel() {
			return sModel;
		}
		
		public String getColor() {
			return sColor;
		}
		
		public String getMatricula() {
			return sMatricula;
		}
		
		public void visualitzar() {
			System.out.println("Marca....: " + sMarca);
			System.out.println("Model....: " + sModel);
			System.out.println("Motor....: " + sColor);
			System.out.println("Matricula: " + sMatricula);
		}
}

File: DAW2/ActivitatsPHP/Exercicis/015_ModelVistaControlador/model/cotxe.php
<?php

class Cotxe 
{
    private $id;
    private $marca;
    private $model;
    private $color;
    private $propietari;

    function __construct($mId, $mMarca, $mModel, $mColor, $mPropietari) {
        $this -> id = $mId;
        $this -> marca = $mMarca;
        $this -> model = $mModel;
        $this -> color = $mColor;
        $this -> propietari = $mPropietari;
    }

    function setId($mId) {
        $this -> id = $mId;
    }

    function setMarca($mMarca) {
        $this -> marca = $mMarca;
    }

    function setModel($mModel) {
        $this -> model = $mModel;
    }

    function setColor($mColor) {
        $this -> color = $mColor;
    }

    function setPropietari($mPropietari) {
        $this -> propietari = $mPropietari;
    }

    function getId() {
        return $this -> id;
    }

    function getMarca() {
        return $this -> marca;
    }

    function getModel() {
        return $this -> model;
    }

    function getColor() {
        return $this -> color;
    }

    function getPropietari() {
        return $this -> propietari;
    }
    
}
?>